subprojects/shipkit/src/test/groovy/org/shipkit/gradle/ShipkitJavaIntegTest.groovy

```diff
@@ -92,6 +91,6 @@ class ShipkitJavaIntegTest extends GradleSpecification {
 :performRelease"""
 
         where:
-            gradleVersionToTest << determineGradleVersionsToTest()
+        gradleVersionToTest << determineGradleVersionsToTest()
```

Comment:
Maybe i would be better to adjust the rule to permit/force indentation in Spock tests? :)

Reply:
yes, we could do so. On the other hand we do have consistent indentation now.

Reply:
Can we try to configure codenarc so that it is possible to have one liner "when:/then:"? 

```Java
when: conf.team.contributors = ["ala:"]
then: thrown(TeamParser.InvalidInput.class)
```

If above it's not possible, I'd say it's better to have consistent whitespace even though it is more restrictive. If can however make use of Spock more convenient, that would be better. @epeee, do you want to take a quick look at it?

Reply:
yes, I will have a look at it.

Reply:
I have disabled the `IndentationRule` for now and reverted the spock label related changes. 
It looks like codenarc master already has some fixes in that area. Let's pick them up once they do a new release and give it another try.